packages/account/src/Components/Routes/route-with-sub-routes.jsx

```diff
@@ -18,9 +19,15 @@ const RouteWithSubRoutes = route => {
         } else if (route.is_authenticated && !route.is_logged_in && !route.is_logging_in) {
             redirectToLogin(route.is_logged_in, getLanguage());
         } else {
-            const default_subroute = route.routes ? route.routes.find(r => r.default) : {};
+            const sub_paths = [];
+            if (route.routes) {
+                route.routes.map(r => r.subroutes.forEach(p => sub_paths.push(p)));
+            }
```

Comment:
Suggesting the use of optional chaining here, plus replacing map with forEach since the return value is not being used. 
```suggestion

                route.routes?.forEach(r => r.subroutes.forEach(p => sub_paths.push(p)));

```

Reply:
Applied albeit modified to combine with amam's suggestions 🙏🏼